modules/experimental-layers/src/utils/gpu-grid-aggregator.js

```diff
@@ -282,104 +508,39 @@ export default class GPUGridAggregator {
       }
       positionsBuffer = new Buffer(gl, {size: 2, data: new Float32Array(positions)});
       weightsBuffer = new Buffer(gl, {size: 1, data: new Float32Array(weights)});
-      this.gridAggregationModel.setAttributes({
+      createPos64xyLow = opts.fp64;
+      Object.assign(aggregationModelAttributes, {
         positions: positionsBuffer,
         weights: weightsBuffer
       });
       this.gridAggregationModel.setVertexCount(positions.length / 2);
       this._setState({positionsBuffer, weightsBuffer});
     }
 
-    if (changeFlags.cellSizeChanged || !gridPixelBuffer) {
-      if (gridPixelBuffer) {
-        gridPixelBuffer.delete();
-      }
-      if (gridTexCoordsBuffer) {
-        gridTexCoordsBuffer.delete();
+    if (createPos64xyLow) {
```

Comment:
Are these fp64 buffers really expensive to create (relative everything else)? Maybe always create these rather than having so much conditional logic? There is already a lot going on here.

Reply:
Haven't measured the perf hit for these, but I will verify and address in followup PR.